src/config/seas5.yml

```diff
@@ -1,6 +1,6 @@
 blob_prefix: seas5/monthly/processed/precip_em_i
 start_date: 1981-01-01
-end_date: 2024-10-30
```

Comment:
How do these dates get updated in the future?

Reply:
In theory these dates should be static, since they're telling us the start and end dates for which we have available COG data.